Resources/Prototypes/Entities/Objects/Specific/Janitorial/spray.yml

```
@@ -99,6 +99,36 @@
   - type: Tag
     tags:
       - Spray
+
+- type: entity
+  parent: SprayBottle
+  id: BorgSprayBottle
+  name: internal spray jet
+  suffix: Filled
+  description: A pressurized vessel for spraying reagents, installed directly into a custodial cyborg. Typically filled with space cleaner for dealing with those nasty spills.
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      spray:
+        maxVol: 200
+        reagents:
+        - ReagentId: SpaceCleaner
+          Quantity: 200
+
+- type: entity
+  parent: MegaSprayBottle
+  id: BorgMegaSprayBottle
+  name: adv internal spray jet
```

Reminder that this should be `advanced internal spray jet`

```suggestion
  name: advanced internal spray jet
```

---

The whole name doesn’t fit in the borg’s hand, I added a period to help show that its abbreviated.